servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -68,13 +68,11 @@ def update_job_labels(id, body, **kwargs):
 
     # Follow API spec
     result = response.json()
-    all_labels = get_job(id).labels
-    if not all_labels:
-        all_labels = {}
+    if not result.get('labels'):
+        result.get('labels').update({})
```

Comment:
Could you clarify when this situation(the returned result under `labels` field is `None`) happens? And why is it necessary to make it as an empty dict`{}` here?